### PR TITLE
Document the web console application launcher

### PR DESCRIPTION
**Pull request by spadgett**

### CODE REVIEW

**Comment by spadgett** [comment]
@ahardin-rh This is a 3.6 feature.

**Review by ahardin-rh — APPROVED** [review]

File: install_config/web_console_customization.adoc [hunk 1]
```diff
@@ -338,6 +338,42 @@ angular
...
+}, {
+  label: "Manage Account",
+  iconClass: "pficon pficon-user",
+  href: "http://example.com/account",
+  tooltip: "Update email address or password."
+}];
+----
+
+. Save the file and add it to the master configuration at
+*_/etc/origin/master/master-config.yml_*:
```
Comment (spadgett) [inline install_config/web_console_customization.adoc]: Everywhere in this file, we use `master-config.yml`, but I believe it's `master-config.yaml`. I can push an update.

Comment (ahardin-rh) [inline install_config/web_console_customization.adoc]: @spadgett Okay, thanks!

**Comment by ahardin-rh** [comment]
To aid in our publishing process, can you please squash the changes into one commit. Thanks!

**Comment by spadgett** [comment]
@ahardin-rh sure, done

**Comment by david-martin** [comment]
@spadgett I gave this a shot locally, and works well.
However, the label doesn't seem to show for me.
Here's my local version using `oc cluster`
```
oc version
oc v3.6.0-rc.0+7d661a2-37
kubernetes v1.6.1+5115d708d7
features: Basic-Auth

Server https://127.0.0.1:8443
kubernetes v1.6.1+5115d708d7
```
image

I'll dig in to see if its just the version I'm using, or i used the wrong key

File: install_config/web_console_customization.adoc [hunk 2]
```diff
@@ -338,6 +338,42 @@ angular
...
+when links are added, appears to the left of the help menu in the masthead.
+
+. Create the configuration scripts within a file (for example,
+*_applicationLauncher.js_*):
+
+[source, javascript]
+----
+// Add items to the application launcher dropdown menu.
+window.OPENSHIFT_CONSTANTS.APP_LAUNCHER_NAVIGATION = [{
+  label: "Dashboard",                    // The text label
```
Comment (david-martin) [inline install_config/web_console_customization.adoc]: This should be `title` instead of `label`

Comment (david-martin) [inline install_config/web_console_customization.adoc]: https://github.com/openshift/origin-web-console/blob/fc9da4e0926fa04f8e284e8ea1b83023e74c767b/app/scripts/extensions/nav/dropdownMobile.js#L16

**Comment by spadgett** [comment]
@david-martin Thanks for catching this, PR updated.

**Review by dtaylor113 — APPROVED** [review]
LGTM, thanks

**Comment by ahardin-rh** [comment]
@jeff-phillips-18 Any comments, or are we okay to merge? Thanks!

**Review by jeff-phillips-18 — APPROVED** [review]

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/web_console_customization.adoc#install-config-web-console-customization[Customizing the Web Console]
|Added a new xref:../install_config/web_console_customization.adoc#web-console-application-launcher[Application Launcher] section.

**Pull request merged**